src/benchmark/relative_speed.rs

```diff
@@ -8,46 +8,61 @@ pub struct BenchmarkResultWithRelativeSpeed<'a> {
     pub result: &'a BenchmarkResult,
     pub relative_speed: Scalar,
     pub relative_speed_stddev: Option<Scalar>,
-    pub is_fastest: bool,
+    pub is_reference: bool,
+    pub is_faster: bool,
```

Comment:
What's the new meaning here? Maybe rename to `is_faster_than_xyz`, with `xyz` replaced by what ever it is faster than? `is_faster_than_reference`?